### PR TITLE
Add TheJesucristoOsAma

### PR DESCRIPTION
**Pull request by HisuianZoroark**

### CODE REVIEW

File: data/mods/ssb/abilities.ts [hunk 1]
```diff
@@ -248,4 +248,23 @@ export const Abilities: {[k: string]: ModdedAbilityData} = {
...
+		name: "The Grace Of Jesus Christ",
+		onResidualOrder: 28,
+		onResidualSubOrder: 2,
+		onResidual(pokemon) {
+			const plates = [
+				'Draco Plate', 'Dread Plate', 'Earth Plate', 'Fist Plate', 'Flame Plate',
+				'Icicle Plate', 'Insect Plate', 'Iron Plate', 'Meadow Plate', 'Mind Plate',
+				'Pixie Plate', 'Sky Plate', 'Splash Plate', 'Spooky Plate', 'Stone Plate',
+				'Toxic Plate', 'Zap Plate',
+			];
```
Comment (KrisXV) [inline data/mods/ssb/abilities.ts:263]:
```suggestion
			const plates = this.dex.items.all().filter(item => item.isPlate);
```

File: data/mods/ssb/random-teams.ts [hunk 2]
```diff
@@ -72,6 +72,12 @@ export const ssbSets: SSBSets = {
 		signatureMove: 'Hat-Trick',
 		evs: {atk: 252, def: 4, spe: 252}, nature: 'Jolly', teraType: 'Any',
 	},
+	TheJesucristoOsAma: {
+		species: 'Arceus', ability: 'The Grace Of Jesus Christ', item: ['Draco Plate', 'Dread Plate', 'Earth Plate', 'Fist Plate', 'Flame Plate', 'Icicle Plate', 'Insect Plate', 'Iron Plate', 'Meadow Plate', 'Mind Plate', 'Pixie Plate', 'Sky Plate', 'Splash Plate', 'Spooky Plate', 'Stone Plate', 'Toxic Plate', 'Zap Plate'], gender: 'N',
+		moves: ['Earthquake', 'Surf', 'Judgment'],
```
Comment (KrisXV) [inline data/mods/ssb/random-teams.ts:77]:
```suggestion
		species: 'Arceus', ability: 'The Grace Of Jesus Christ', gender: 'N',
		item: [
			'Draco Plate', 'Dread Plate', 'Earth Plate', 'Fist Plate', 'Flame Plate', 'Icicle Plate', 'Insect Plate', 'Iron Plate', 'Meadow Plate',
			'Mind Plate', 'Pixie Plate', 'Sky Plate', 'Splash Plate', 'Spooky Plate', 'Stone Plate', 'Toxic Plate', 'Zap Plate',
		],
		moves: ['Earthquake', 'Surf', 'Judgment'],
```

File: data/mods/ssb/random-teams.ts [hunk 3]
```diff
@@ -72,6 +72,12 @@ export const ssbSets: SSBSets = {
...
+		signatureMove: 'The Love Of Christ',
+		evs: {hp: 4, spa: 252, spe: 252}, nature: 'Timid',
```
Comment (KrisXV) [inline data/mods/ssb/random-teams.ts:79]: we need to hardcode arceus to not be able to terastallize in `canTerastallize`

Comment (HisuianZoroark) [reply data/mods/ssb/random-teams.ts:83]: Fixed

**Pull request merged**